### PR TITLE
Added model HG07834C

### LINKED ISSUE #3280: New Livarno/Lidl RGB bulbs not supported
Just purchased some new Lidl RGB bulbs and the manufacturer/model doesn't seem to be recognised:

> Zigbee2MQTT:warn  2021-11-04 09:58:13: Device '0xb4e3f9fffec0a317' with Zigbee model 'TS0505B' and manufacturer name '_TZ3000_qd7hej8u' is NOT supported, please follow https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html

Will follow tutorial tonight but if someone wants to add this already to the dev branch I'd appreciate.

If there's anything I can help with please let me know.

### PR DESCRIPTION
**Pull request by mribeiro**
Copide from model HG06106C. Tested and works as expected.
New Lidl RGB bulbs ship in a smaller box.
Manufacturer name: _TZ3000_qd7hej8u
Model ID TS0505B

Fixes #3280

### CODE REVIEW

File: devices/lidl.js [hunk 1]
```diff
@@ -155,6 +155,17 @@ module.exports = [
             device.getEndpoint(1).saveClusterAttributeKeyValue('lightingColorCtrl', {colorCapabilities: 29});
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0505B', manufacturerName: '_TZ3000_qd7hej8u'}],
+        model: 'HG07834C',
+        vendor: 'Lidl',
+        description: 'Livarno Lux E27 bulb RGB',
+        ...extend.light_onoff_brightness_colortemp_color({disableColorTempStartup: true}),
```
Comment (Koenkk) [inline devices/lidl.js:163]: Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

Comment (mribeiro) [reply devices/lidl.js:163]: Added to the exception, just like HG07834C as they are very alike

Comment (Koenkk) [reply devices/lidl.js:163]: What results do you get when reading the min/max color temp? I prefer not to add new devices to the exceptions list.

Comment (mribeiro) [reply devices/lidl.js:163]: I got `Read result of 'lightingColorCtrl': {"colorTempPhysicalMin":153,"colorTempPhysicalMax":500}`

And I have it set in the code now as 

```
...extend.light_onoff_brightness_colortemp_color({disableColorTempStartup: true, colorTempRange: [153, 500]}),
```

But I'm wondering if I should keep the `disableColorTempStartup` . How can I know this?

Comment (Koenkk) [reply devices/lidl.js:163]: Please update the pr with this code, these bulbs do not support setting the color temp startup therefore it is disabled

Comment (mribeiro) [reply devices/lidl.js]: That's done! Thanks for the help!

**Pull request merged**